### PR TITLE
Display user friendly errors

### LINKED ISSUE #271: Error messages in HEDA Settings page should be nicely displayed
In a user-friendly manner. Now they are just shown as a string at the bottom of the page.

### PR DESCRIPTION
**Pull request by davidjray**
# Critical Changes

# Changes
- Errors in HEDA Settings will display in a notification toast.
# Issues Closed
Fixes #271

### CODE REVIEW

**Comment by davidjray** [comment]
Error display:
screen shot 2016-12-30 at 4 13 11 pm

**Comment by ceiroa** [comment]
@davidjray Could you please also add an screenshot of how it looks in Classic? Thanks!

**Comment by davidjray** [comment]
@ceiroa Good idea!

Here it is in Classic:
screen shot 2017-01-03 at 9 12 01 am

File: src/aura/STG_CMP_Container/STG_CMP_Container.cmp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	<div class="slds-notify_container slds-hide" aura:id="notificationToast">
+		<div class="slds-notify slds-notify--toast slds-theme--error" role="alert">
+			<span class="slds-assistive-text">Error</span>
+			<lightning:buttonIcon iconName="utility:close" variant="bare" onclick="{! c.hideToast }" alternativeText="Close" class="slds-icon slds-notify__close slds-button--icon-inverse" size="large" />
+			<div class="slds-notify__content slds-grid slds-align-middle">
+
+				<lightning:icon iconName="utility:warning" class="slds-icon slds-m-right--small slds-col slds-no-flex" size="medium" />
+
+				<div class="slds-col slds-align-middle">
+					<!-- By Using the ID "auraErrorMessage" the system will automatically append the error message to the element-->
```
Comment (ceiroa) [inline src/aura/STG_CMP_Container/STG_CMP_Container.cmp]: Thanks for leaving this comment!

File: src/aura/STG_CMP_Container/STG_CMP_ContainerHelper.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+({
+	showToast: function(component, event, helper, message) {
+		console.log("TOASTEr!");
```
Comment (ceiroa) [inline src/aura/STG_CMP_Container/STG_CMP_ContainerHelper.js]: Can we remove this log message, just to clean up?

**Comment by davidjray** [comment]
@ceiroa The display of the messages as text at the bottom was the default behavior. Lightning Out (for VisualForce) was adding those messages as text by default. I am listening to and acting upon the system event "aura:systemError" and showing a error message inside a container that has the ID "auraErrorMessage". Aura automatically appends the error message to that DIV. By using the system message event the default behavior is bypassed since the framework is aware I am subscribed to "aura:systemError".

File: src/aura/STG_CMP_Container/STG_CMP_ContainerHelper.js [hunk 3]
```diff
@@ -1,7 +1,5 @@
 ({
 	showToast: function(component, event, helper, message) {
-		console.log("TOASTEr!");
-
```
Comment (davidjray) [inline src/aura/STG_CMP_Container/STG_CMP_ContainerHelper.js]: @ceiroa Removed 👍

**Pull request merged**